jdaviz/configs/cubeviz/plugins/viewers.py

```diff
@@ -25,3 +25,10 @@ def data(self, cls=None):
                 for layer_state in self.state.layers
                 if hasattr(layer_state, 'layer') and
                 isinstance(layer_state.layer, BaseData)]
+
+
+# TODO: Officially deprecate this with coordination with JDAT notebooks team.
+# For backward compatiblity only.
+class CubeVizImageView(CubevizImageView):
```

Comment:
Actually, I am not sure if this trick would work here given the `@viewer_registry` usage.

If the user does not call this class directly, maybe we can just rename without any backward compatibility?